src/main/java/com/openlattice/graph/core/GraphService.java

```diff
@@ -51,9 +51,9 @@
 
     Stream<Edge> getEdgesAndNeighborsForVertex( UUID entitySetId, UUID vertexId );
 
-    Stream<Edge> getEdgesAndNeighborsForVertices( UUID entitySetId, EntityNeighborsFilter filter );
+    Stream<Edge> getEdgesAndNeighborsForVertices( EntityNeighborsFilter filter );
```

Comment:
What's the reason for removing entitySetId as a parameter here and below?

Reply:
Because the `EntityNeighborsFilter` now includes the entitySetId:
Before it was: `entityKeyIds: Set<UUID>` and now it's `entityKeyIds: Map<UUID, Set<UUID>>` where the keys of the map are the entity set ids.

Reply:
Ty for clarifying!